pkg/controller/sidecar/sync.go

```diff
@@ -33,7 +31,7 @@ func (c *Controller) decommissionNode(ctx context.Context, svc *corev1.Service)
 		return err
 	}
 
-	opMode, err := scyllaClient.OperationMode(ctx, c.hostAddr)
+	opMode, err := scyllaClient.OperationMode(ctx, localhost)
```

Comment:
would it be better to use the "localhost" DNS/alias that's IP protocol independent?

Reply:
done